README.md

```diff
@@ -53,8 +53,7 @@ Impactful scientific papers about GPT-4 and its predecessors.
 - [doctorgpt](https://github.com/ingyamilmolinar/doctorgpt) - Production log error diagnosing
 - [MusicGPT](https://github.com/d3n7/GPT-4-To-MIDI) - Make polyphonic music in the form of MIDI files with GPT-4 or 3.5.
 - [StockGPT](https://github.com/d3n7/StockGPT) - Primitive technique to predict stock movements with GPT-4 or 3.5.
-
-
+- [Haddock](https://www.haddock.ai/) - Search GPT-4 Generated Scripts for Gaming Engines for free. Currently supporting Roblox, Unity, and Unreal Engine. 
```

Comment:
Try to make the description shorter and not sound like an advertisement.
For example: "Search GPT-4 generated scripts for gaming engines - Roblox, Unity, and Unreal."

Reply:
I also recommend linking the github repository instead of your website as the rest of the OSS entries do.

Reply:
@radi-cho sure, done!